### ISSUE TITLE
Suggestion: RoPS computation using OpenMP 

### ISSUE THREAD

**Issue by 11pguillon** [PointCloudLibrary/pcl#2628]
ROPSEstimationOMP ?
There is currently no OpenMP support for RoPS feature computation. The main loop over feature points is compatible with multithreading, so I suggest it could be added as it is already for SHOT and FPFH.

**Comment by taketwo** [comment]
Lately we are trying to avoid class duplication and incorporate OpenMP extensions directly into the base class.  See [`MovingLeastSquares`](https://github.com/PointCloudLibrary/pcl/blob/master/surface/include/pcl/surface/mls.h) for an example. The class exposes a method to set the number of threads and in the implementation file the OpenMP-related stuff is `#ifdef`ed.

**Comment by larshg** [comment]
@shrijitsingh99 Maybe one for you here?

**Comment by shrijitsingh99** [comment]
Yup, have noted it down. Once I am done with the existing OMP implementations, will look into adding it for this.
